chirp/drivers/tdh8.py

```diff
@@ -440,7 +440,8 @@
 GMRS_FREQ_A = GMRS_FREQS1 + GMRS_FREQS3
 
 HAM_GMRS_FREQS = [162.55000, 162.40000, 162.47500, 162.42500, 162.45000,
-                  162.50000, 162.52500, 161.65000, 161.77500, 161.75000]
+                  162.50000, 162.52500, 161.65000, 161.77500, 161.75000,
+                  162.00000]
```

Comment:
Can you please use the values already in `bandplan_na`?

Reply:
Here you may need to define it yourself, because I can't find these values ​​in bandplan_na.

Reply:
Oh, I didn't even look at the frequencies. These are NOAA weather channels, not `HAM_GMRS_FREQS` so the variable is named incorrectly. But, you should store them in Hz, like in `bandplan_na`. Really we should add these NOAA channels to `bandplan_na` but I won't make you do that. Just store them as `162550000` and not `162.55`. Floating point numbers do not have infinite resolution and are not exact.